### DISCUSSION TITLE
Nervos CKB Support?

### DISCUSSION THREAD

**Discussion by EsloTech** [coinfoundry/miningcore#1649] [General]
Hi there, is there any plan to support ckb? or would I be able to add support for ckb myself?

Thank you!

**Comment by konstiantyn** [comment]
You can refer follow links.

https://github.com/nervosnetwork/ckb
https://github.com/insight-nervos/btcpool-docker-compose

**Reply by EsloTech** [reply]
Thanks! Yes, I actually have https://github.com/insight-nervos/btcpool-docker-compose installed but code is not updated and I have some issues with the JSONRPC and database for get_block - 
It is possible to implement in miningcore based on this fork?

**Comment by konstiantyn** [comment]
Of course, Yes.

**Reply by EsloTech** [reply]
Do you have any guideline to follow?

**Reply by konstiantyn** [reply]
You have to add algorithm first
https://github.com/nervosnetwork/rfcs/blob/master/rfcs/0010-eaglesong/0010-eaglesong.md

**Reply by EsloTech** [reply]
I will take a look, I was so close to get working the btcpool... thanks buddy

**Reply by matt-nervos** [reply]
> You have to add algorithm first https://github.com/nervosnetwork/rfcs/blob/master/rfcs/0010-eaglesong/0010-eaglesong.md

in C?

**Reply by EsloTech** [reply]
@konstiantyn lets is know please!

**Reply by sudonanohash** [reply]
> @konstiantyn lets is know please!

did you get any luck doing it?
